.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Create Release
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - '[0-9].[0-9]+.[0-9]+' # Push events to any matching semantic tag. For example, 1.10.1 or 2.0.0.
```

Comment:
We have a tag `1.0.0-alpha` that wouldn't work with this.  What about:

```
- '[0-9]+.[0-9]+.[0-9]*?''
```

Not sure if that's enough, but I'm trying to add things like `-alpha` on the end.

Reply:
True. It's a pity GitHub does not allegedly support full regex syntax. I will try to use the regex specified under server docs first to see if it might actually work. If not, we should at least modify it slightly

Reply:
As suspected, [the recommended regex](https://github.com/semver/semver/blob/master/semver.md#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string) is not supported in YAML. How about we use `[0-9]+.[0-9]+.[0-9]+-?[a-z]+`? @humphd

Reply:
Apparently, neither `'[0-9]+.[0-9]+.[0-9]*?'` nor `'[0-9]+.[0-9]+.[0-9]+-?[a-z]+'` are correct. Regex works very crooky in YAML. Needs to investigate more... @humphd

Reply:
For some reason `[0-9]+.[0-9]+.[0-9]+-?[a-z]*` [does not get triggered by tag like `1.5.4`](https://github.com/birtony/telescope/releases/tag/1.5.4). It seems to be a valid expression in YAML though, as [it works for tags like `1.5.4-alpha`](https://github.com/birtony/telescope/releases/tag/1.5.4-alpha). @humphd any idea why is it happening?

Reply:
I think we could start with a simple check for x.y.z style semver, and file an issue to fix this later.  It doesn't need to block 1.6.

Reply:
No idea.  Let's do the most simple check possible on the tag and get this in, then refine it in follow-ups.

Reply:
Agreed.